lighthouse-core/gather/gather-runner.js

```diff
@@ -64,8 +64,13 @@ class GatherRunner {
    * @param {number=} duration
    * @return {!Promise}
    */
-  static loadBlank(driver, url = 'about:blank', duration = 300) {
-    return driver.gotoURL(url).then(_ => new Promise(resolve => setTimeout(resolve, duration)));
+  static loadBlank(driver, url) {
+    // The real about:blank doesn't fire onload and is full of mysteries
+    //   https://github.com/whatwg/html/issues/816#issuecomment-288931753
+    // To improve speed and avoid anomalies, we use a basic data uri blank page
+    url = url || 'data:text/html,<!doctype html><title>Resetting Page...</title>';
```

Comment:
I actually kind of love that WPT's is a non-white color to signal when Chrome gets the first byte by turning white but not sure if that will cause issues :)